ManatiCR/manatime
# 11: Submit timesheet reminder.

**Steps to test**
- Go to: `/admin/config/manatime/timesheet-reminder`, and see if the form is displayed. Once there save the time the reminders will be sended (The actual time).
- Run the Cron.
- Since the terminal run `vagrant ssh` and go to: `/tmp/devel-mails` once there run `l` to see if the mails were created succesfully.

**DoD**
- UI Config Form (with custom permission) to select Day of Week and Hour of Day to send reminder (explain in form that the time is approximate)
- Configure a cron task that run only for specified time that sents a message (using message + message notify + ¿rules?) to all collaborators who haven't submitted the timesheet for current (recently past) week.

[Issue link](https://github.com/ManatiCR/manatime/issues/11)
